### ISSUE TITLE
Police baton east-facing sprite is backwards

### ISSUE THREAD

**Issue by Zilenan91** [tgstation/tgstation#14844]
Some of the other sprites with this might be bugged too.

**Comment by Zilenan91** [comment]
This only happens while in the right hand, strangely enough.

**Comment by WJohn** [comment]
Someone just screwed up the inhands. Super easy fix. Grab the west facing sprite from the left hand, put it on the east facing sprite for the right hand, and press the swap button. Done.

**Comment by PKPenguin321** [comment]
@WJohn I had a look at this and it turns out pretty much all the sprites for both hands except for the south ones are bad. The north ones are in the correct hands but overlay over them instead of appearing in the hand (also it's just the south sprite flipped), east and west are literally copied from the north and south sprites so they don't fit in the hands at all.
basically this needs completely new/severely tweaked inhands for every direction.

in this image, the top four humans are lefthanded and bottom four are righthanded:
![bad inhands](http://puu.sh/mIfPg/37884195c9.png)
as you can see, the south sprites both look fine.
the north sprites go over the back of the hand, which they should not. it makes them seem to be hovering.
the east and west sprites for the left hand are both hovering and being held at about crotch level.
the east and west sprites for the right hand are both hovering and coming from the weilder's asshole.

tl;dr: we need somebody that can actually sprite to fix this

**Comment by WJohn** [comment]
Oh nevermind me then. I went under the assumption that only the reported thing was bad. Looks like everything there is awful though.

**Comment by LanCartwright** [comment]
i swear i though tthis was intended because the police baton is technically a tonfa

**Comment by PKPenguin321** [comment]
Fixed by https://github.com/tgstation/tgstation/pull/18098
